src/cocotb/types/logic_array.py

```diff
@@ -273,6 +287,37 @@ def from_signed(cls, value: int, range: Range) -> "LogicArray":
             )
         return LogicArray(value, range=range)
 
+    @classmethod
+    def from_bytes(
+        cls,
+        value: Union[bytes, bytearray],
+        range: Optional[Range] = None,
+        byteorder: str = "big",
```

Comment:
Running mypy on this says this should be `Literal["big"] | Literal["little"]`. I should be able to use a `if TYPE_CHECKING` block to do the import since this is a Python 3.8+ feature.

Reply:
Yes, that's another option. Not really of highest importance, but nice to have.